cf/scf-develop.yaml.erb

```diff
@@ -266,6 +275,27 @@ jobs:
         params:
           file: out/version
 
+  - name: package-bits
+    plan:
+      - aggregate:
+        - get: src
+          passed:
+          <% if manual_pull_sources %>
+          - pull-sources
+          <% end %>
+          trigger: true
+        - get: ci
+        - get: s3.certstrap-binary
```

Comment:
Done with commit 122499aa576beda015c221ab2c7763a330041d1c
Note, this has an adjunct PR in SCF, see https://github.com/SUSE/scf/pull/1044